java/impl/aws/src/main/java/co/worklytics/psoxy/Handler.java

```diff
@@ -9,21 +9,20 @@
 import com.amazonaws.services.lambda.runtime.Context;
 import com.amazonaws.services.lambda.runtime.events.APIGatewayV2HTTPEvent;
 import com.amazonaws.services.lambda.runtime.events.APIGatewayV2HTTPResponse;
-import com.google.common.annotations.VisibleForTesting;
 import lombok.SneakyThrows;
 import lombok.extern.java.Log;
-import org.apache.commons.codec.binary.Base64;
-import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.lang3.exception.ExceptionUtils;
 import org.apache.commons.lang3.tuple.Pair;
 import org.apache.http.HttpHeaders;
 
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.nio.charset.StandardCharsets;
-import java.util.Optional;
-import java.util.zip.GZIPOutputStream;
+import static co.worklytics.psoxy.ResponseCompressionHandler.isCompressionRequested;
```

Comment:
Just personal taste, I'd prefer to not import it and directly use `ResponseCompressionHandler.isCompressionRequested(...)` on L68